tools/Julia/common.jl

```diff
@@ -43,7 +43,7 @@ end
 function log_gamma_distrib(a, p)
   out = 0.25 * p * (p - 1) * 1.1447298858494002 #convert(Float64, log(pi))
 	for j in 1:p
-    out += lgamma(a + 0.5*(1 - j))
+    out += (logabsgamma(a + 0.5*(1 - j)))[1]
```

Comment:
Does Zygote have an adoint for this?   If not, add lgamma' = digamma

Reply:
I mean, if it does have an adjoint for lgamma, and not one for logabsgamma.

Reply:
Actually, it's fine -- wait until we hit it.  I.e. all lgtm,.

Reply:
It seems to run fine so I assume Zygote has already implemented it.